Lib/asyncio/taskgroups.py

```diff
@@ -185,7 +186,7 @@ def _on_task_done(self, task):
         if self._parent_task.done():
             # Not sure if this case is possible, but we want to handle
```

Comment:
I think this is not possible because `        self._errors.append(exc)` on line 182 will fail first with `AttributeError: 'NoneType' object has no attribute 'append'`